src/aks-preview/azext_aks_preview/__init__.py

```diff
@@ -31,7 +31,12 @@ def load_command_table(self, args):
         return self.command_table
 
     def load_arguments(self, command):
-        super(ContainerServiceCommandsLoader, self).load_arguments(command)
+        from sys import version_info
+        if version_info[0] < 3:
+            super(ContainerServiceCommandsLoader, self).load_arguments(command)
```

Comment:
This syntax works on Python 3 as well. There's no need for this if statement.

Reply:
Are you sure about that? Since my teammate and I have problem to use `super(ContainerServiceCommandsLoader, self).load_arguments(command)`, It failed with 
```
The command failed with an unexpected error. Here is the traceback:

super(type, obj): obj must be an instance or subtype of type
Traceback (most recent call last):
  File "/usr/local/Cellar/azure-cli/2.0.63/libexec/lib/python3.7/site-packages/knack/cli.py", line 206, in invoke
    cmd_result = self.invocation.execute(args)
  File "/usr/local/Cellar/azure-cli/2.0.63/libexec/lib/python3.7/site-packages/azure/cli/core/commands/__init__.py", line 445, in execute
    self.commands_loader.load_arguments(command)
  File "/usr/local/Cellar/azure-cli/2.0.63/libexec/lib/python3.7/site-packages/azure/cli/core/__init__.py", line 268, in load_arguments
    loader.load_arguments(command)  # this adds entries to the argument registries
  File "/Users/qizhe/cli-extension/azure-cli-extensions/src/aks-preview/azext_aks_preview/__init__.py", line 34, in load_arguments
    super(ContainerServiceCommandsLoader, self).load_arguments(command)
TypeError: super(type, obj): obj must be an instance or subtype of type
```, so I decide to use the if else statement to unblock us.

Reply:
Nowhere else in the CLI or any extension does anyone do this, so the true problem is likely something else. That being said, I have no problem merging this, but it shouldn't be necessary.